spec/indexers/work_indexer_spec.rb

```diff
@@ -290,6 +290,77 @@
       end
     end
 
+    describe "with OCR" do
+      let(:assets) do
+        [3,2,1].map { |page| create(
+          :asset,
+          position: page,
+          published: true,
+          hocr: hocr
+          ) }
+      end
+
+      let (:hocr) { File.read('spec/test_support/hocr_xml/hocr.xml')}
+      let(:expected_hocr) {
+        ["CAUTION All units must be connected as above before the Power Supply is connected."] * 3
+      }
+
+      # transcription : en
+      # translation   : en
+      let(:english_only_work) do
+        create(:public_work, language: ['English'], members: assets )
+      end
+
+      # transcription : agnostic
+      # translation   : en
+      let(:bilingual_work) do
+        create(:public_work, language: ['English', 'German'], members: assets)
+      end
+
+      # transcription : de
+      # translation   : en
+      let(:german_only_work) do
+        create(:public_work, language: ['German'], members: assets )
+      end
+
+      let(:nil_members_work) do
+        create(:public_work, members: [])
+      end
+
+      before do
+        allow(nil_members_work).to receive(:members).and_return(nil)
+      end
+
+      let(:output_hash) { WorkIndexer.new.map_record(language_test_work) }
+      let(:english) { output_hash["searchable_fulltext_en"] }
+      let(:german)  { output_hash["searchable_fulltext_de"] }
+      let(:unsure)  { output_hash["searchable_fulltext_language_agnostic"] }
+      describe "text known to be in English" do
+        let(:language_test_work) { english_only_work }
+        it "goes in searchable_fulltext_en" do
+          expect(english).to eq(expected_hocr)
+          expect(german).to be_nil
+          expect(unsure).to be_nil
+        end
+      end
+      describe "text known to be in German" do
+        let(:language_test_work) { german_only_work }
+        it "goes in searchable_fulltext_de" do
+          expect(german).to eq(expected_hocr)
+          expect(english).to be_nil
+          expect(unsure).to be_nil
+        end
+      end
+      describe "text in either English or German or something else" do
```

Comment:
name this what it actually is. Is it in "English or German or something else"? (not even sure what that is supposed to mean!)  Looks like it's a bilingual work, so name the block that?